README.md

```diff
@@ -1,2 +1,109 @@
 # Conduit Connector Salesforce
 
+## General
+
+The Salesforce plugin is one of [Conduit](https://github.com/ConduitIO/conduit) plugins.
+It currently provides only source Salesforce connector, allowing for receiving Salesforce changes events in a Conduit pipeline.
+
+## How to build it
+
+Run `make`.
+
+## Source
+
+The Source connector subscribed to given topic and listens for events published by Salesforce.
+
+### How it works
+
+This section describes the technical details of how the connector works.
+
+#### Authentication
+
+The connector authenticates with Salesforce using OAuth credentials. Once successfully authenticated, Salesforce returns:
+- `Client ID` - token used in Streaming API requests.
+- `Instance URL` - Streaming API base URL for given credentials.
+
+#### Streaming API
+
+Salesforce uses [CometD](https://docs.cometd.org/current7/reference/#_preface) as a streaming protocol. More details can be found in link below and in References:
+https://developer.salesforce.com/docs/atlas.en-us.api_streaming.meta/api_streaming/using_streaming_api_client_connection.htm
+
+#### Push Topics
+
+Connector is capable of listening to [custom Push Topics](https://developer.salesforce.com/docs/atlas.en-us.api_streaming.meta/api_streaming/code_sample_interactive_vfp_create_pushtopic.htm).
+All types of notifications are supported: `NotifyForOperationCreate`, `NotifyForOperationUpdate`, `NotifyForOperationUndelete`, `NotifyForOperationDelete`.
+
+#### Data types
```

Comment:
Even though it might be obvious from the previous paragraph, I believe it would be good to mention which which Salesforce data types can be pulled (I guess it's only Salesforce objects, and not metadata, and it's done through the topic, not through the connector itself?).

Reply:
I'm fine with that or anything else which helps someone using the connector for the first time, figure out what the connector can (or cannot) do.